app/models/comment.rb

```diff
@@ -142,4 +142,11 @@ def answer_comment_notify(current_user)
     notify_users(uids, current_user)
     notify_tag_followers(already + uids)
   end
+
+  def comment_spam
```

Comment:
Oh cool! Let's name this just `def spam` to match node.rb, and add a corresponding `def publish` method? 

https://github.com/publiclab/plots2/blob/fa16a69f8f475c101bc23325f4df7411c2172872/app/models/node.rb#L151-L161

How does that sound? Then we can unit test this and functional test the above. This is looking great!!!

Reply:
will do